estate/controllers/main.py

```diff
@@ -0,0 +1,10 @@
+from odoo import http
+from odoo.http import request
+
+class MyController(http.Controller):
+    
+    @http.route('/my/custom/route', auth='public', website=True)
```

Comment:
```suggestion
    @http.route('/my/custom/route', auth='public', website=True)
```
what this website=True will do?